ckanext/depositar_theme/helpers.py

```diff
@@ -67,3 +72,43 @@ def send_reg_link(user):
     subject = subject.split('\n')[0]
 
     mail_user(user, subject, body)
+
+@core_helper
+def get_format_count():
+    results = logic.get_action('package_search')({}, {})['results']
```

Comment:
優先使用 `tk.get_action()`

```suggestion
    results = tk.get_action('package_search')({}, {})['results']
```